crates/frontend/src/components/dimension_form.rs

```diff
@@ -165,7 +165,7 @@ where
                             <textarea
                                 type="text"
                                 placeholder="JSON schema"
-                                class="input input-bordered mt-5 rounded-md resize-y w-full max-w-md"
+                                class="input input-bordered mt-5 rounded-md resize-y w-full max-w-md pt-3"
```

Comment:
`nitpick`
mt-3 would be better

Reply:
pt-3 is for the inner padding of the `textarea`, So that it looks cleaner, there is already a `mt` applied to it.